integrations/pom.xml

```diff
@@ -56,7 +58,6 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>mailer</artifactId>
-      <version>1.20</version>
```

Comment:
This shouldn’t be showing up in this PR, can you merge with master again please?

Reply:
Yeah sure.